### PR TITLE
06-loops-and-functions.md: update keypoints

### PR DESCRIPTION
**Pull request by wrightaprilm**

### CODE REVIEW

File: _episodes/06-loops-and-functions.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ objectives:
     - "Build reusable code in Python."
     - "Write functions using conditional statements (if, then, else)."
 keypoints:
-    - "FIXME"
+    - "Loops help automate repetitive tasks over sets of items."
+    - "Loops can be combined with functions we have learned so far to make data analysis and processing more efficient."
+    - "Conditional statements allow us to sort data and perform different operations on different subsets of the data."
+    - "Functions can enable code reuse."
```
Comment (maxim-belkin) [inline _episodes/06-loops-and-functions.md]: Functions  enable code reuse

File: _episodes/06-loops-and-functions.md [hunk 2]
```diff
@@ -12,7 +12,10 @@ objectives:
     - "Build reusable code in Python."
     - "Write functions using conditional statements (if, then, else)."
 keypoints:
-    - "FIXME"
+    - "Loops help automate repetitive tasks over sets of items."
+    - "Loops can be combined with functions we have learned so far to make data analysis and processing more efficient."
+    - "Conditional statements allow us to sort data and perform different operations on different subsets of the data."
```
Comment (maxim-belkin) [inline _episodes/06-loops-and-functions.md]: Conditional statements enable execution of different operations on different data.

File: _episodes/06-loops-and-functions.md (hunk 1 above)
Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]:
```suggestion
    - "Functions enable code reuse."
```

File: _episodes/06-loops-and-functions.md [hunk 3]
```diff
@@ -12,7 +12,10 @@ objectives:
     - "Build reusable code in Python."
     - "Write functions using conditional statements (if, then, else)."
 keypoints:
-    - "FIXME"
+    - "Loops help automate repetitive tasks over sets of items."
+    - "Loops can be combined with functions we have learned so far to make data analysis and processing more efficient."
```
Comment (maxim-belkin) [inline _episodes/06-loops-and-functions.md]: ....Um, not sure if we need this one, but:

Loops and functions are fundamental to data analysis.

(???)

File: _episodes/06-loops-and-functions.md (hunk 2 above)
Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]:
```suggestion
    - "Conditional statements enable execution of different operations on different data."
```

File: _episodes/06-loops-and-functions.md (hunk 3 above)
Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]: I think that belabors the point slightly, no?

Comment (maxim-belkin) [reply _episodes/06-loops-and-functions.md]: Slightly? :)

I'm not sure what `make data analysis [snip] efficient` means...

Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]: What I'm really trying to get at is that loops are versatile and can be combined with all the junk they learned before this lesson to do more science in less time.

Comment (maxim-belkin) [reply _episodes/06-loops-and-functions.md]: > What I'm really trying to get at is that loops ... [snip] to do more science in less time

I think keypoints have to be about concrete stuff that is covered in the lesson, not a summary. I suggest that we either remove this keypoint or rephrase it using the text that you used above (without the "junk" part :) ).  BUT, given the time constraints imposed on us, feel free to ignore this comment and merge it as is.

Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]:
```suggestion
    - "Loops can be combined with functions we have learned so far to process data more efficiently than can be accomplished by hand."
   ```
```

Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]: Maybe like so?

Comment (maxim-belkin) [reply _episodes/06-loops-and-functions.md]: I'd get rid of `can be` and `we have learned so far` 

Loops combined with functions provide a way to...

Comment (wrightaprilm) [reply _episodes/06-loops-and-functions.md]: brevity is the soul of wit. 

Not sure what that makes me ;)

**Pull request merged**